### PR TITLE
Update Test Suite

### PR DESCRIPTION
**Pull request by MohamedSadiq102**
- Amend whitespace in config.
- Add switch to docker compose set up.
- Comment failing test and link to github issue
- Ensure test data is purged at the end of each test
-Edit the Link of context 
- Changes for Notifications-URL

### CODE REVIEW

File: common.js [hunk 1]
```diff
@@ -9,7 +9,7 @@
 const endpoint = process.env.TEST_ENDPOINT || 'http://localhost:1026';
 
 const accEndpoint = process.env.ACC_ENDPOINT || 'http://localhost:3000';
-const notifyEndpoint = process.env.NOTIFY_ENDPOINT || `${accEndpoint}/acc`;
+const notifyEndpoint = process.env.NOTIFY_ENDPOINT || 'http://host.docker.internal:3000/acc';
```
Comment (kzangeli) [inline common.js:12]: So, you've decided to hardcode the receiver of notifications?
Might be a good idea.
Make sure you change this everywhere.
Also, the env var for the receiver I think is part of the README. If so, it should be removed from there.

Comment (kzangeli) [reply common.js:12]: I checked.
README.md says this:

```
export NOTIFY_ENDPOINT=<The notification endpoint>. Example: `http://host.docker.internal:3000/acc`
```

Comment (MohamedSadiq102) [reply common.js:12]: I have changed it in README and I hope every thing is good now.

fixed : 4c28fa8

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**